packages/config/src/projects/layer2s/index.test.ts

```diff
@@ -92,6 +92,25 @@ describe('layer2s', () => {
       }
     })
 
+    describe('every escrow sinceTimestamp is greater or equal to chains minTimestampForTvl', () => {
+      for (const layer2 of layer2s) {
+        for (const escrow of layer2.config.escrows) {
+          const chain = chains.find((c) => c.name === escrow.chain)
+
+          it(`${layer2.id.toString()} : ${escrow.address.toString()}`, () => {
+            assert(
+              chain?.minTimestampForTvl,
```

Comment:
assert for chain and chain.minTimestampForTvl with appropriae message (see tokens.test.ts), this will be helpful for later debugging